requirements.txt

```diff
@@ -1,2 +1,3 @@
+boto3
```

Comment:
Are you saying that `boto3` instead belongs in `requirements-dev.txt` (or `requirements-test.txt`) and not in `requirements.txt`?  I need to ensure `boto3` is installed somewhere in my pyenv for local testing.  Or are you saying it doesn't belong in any of those places?

Reply:
Why do you need `boto3` in the Python environment?

Reply:
Often tasks such as pushing to or pulling from S3 are delegated to `localhost`, so that the user's AWS creds () when testing locally) or GH Actions' (when testing in GH Actions) can be used.  I assume that is what is happening here.

Reply:
I was mainly thinking about when I had tested portions of the lamdba code locally and I needed to make AWS calls.  Is everyone cool with me just moving `boto3` from `requirements.txt` into `requirements-dev.txt`?

Reply:
I guess my question is do we need `boto3` for continued, regular development of this project? Should future development efforts expect to need `boto3` to work on this project?

Reply:
I contend that it's handy to include it and doesn't really hurt anything, but I will gladly follow the will of the people here.

Reply:
I guess my thinking is you could spin up the Docker image created with this configuration for testing to mimic the environment it would run in when deployed.

Reply:
I'm fine with it going into `requirements-dev.txt` if a comment is added there explaining why it is being added.

Reply:
> I'm fine with it going into `requirements-dev.txt` if a comment is added there explaining why it is being added.

@mcdonnnj Does that work for you?

Reply:
Yes with an explanatory comment that is fine for me.

Reply:
Have a look at https://github.com/cisagov/publish-egress-ip-lambda/pull/4/commits/6de01f31e5a227cfd0937ec8cc988f96b63abc64 and let me know if that works for you.